jetty/src/main/java/com/google/musicanalysis/jettymain/Main.java

```diff
@@ -0,0 +1,67 @@
+/*
```

Comment:
This file was adapted from [here](https://github.com/GoogleCloudPlatform/java-docs-samples/blob/master/appengine-java11/appengine-simple-jetty-main/src/main/java/com/example/appengine/demo/jettymain/Main.java). It executes the WAR file, which is a type of archive that contains our Servlets.